src/graphql/main/schema.graphql

```diff
@@ -567,30 +568,49 @@ input AccountApiKeyDisableInput {
 
 type UserUpdateLanguagePayload {
   errors: [Error!]!
-  user: UserDetails
+  user: UserLanguageDetails
```

Comment:
not sure if user is the proper name for this attribute. UserLanguageDetails only has id and language

Reply:
It's a partial record for a user, so I think that's ok